action.yml

```diff
@@ -1,13 +1,12 @@
-name: 'Hello World'
-description: 'Greet someone and record the time'
+name: 'Cancel workflow action'
+description: 'Cancel previous workflows'
 inputs:
-  who-to-greet:  # id of input
-    description: 'Who to greet'
+  workflow_id:  # id of input
+    description: 'The id of your workflow'
```

Comment:
I have no idea how this was working before 🤔 

We also need `access_token` like its defined in [main.yml](https://github.com/styfle/cancel-workflow-action/blob/master/.github/workflows/main.yml) since the default access token doesn't have permission to cancel workflows.

Reply:
Nice I'll add this too ✅